WTWSMS/common/minor_titles/00_minor_titles.txt

```diff
@@ -2743,16 +2697,16 @@ title_guru = {
 	message = yes
 }
 title_genghis = {
-	dignity = 10
+	dignity = 500
 	revoke_allowed = no
 	
 	realm_in_name = no
 	show_as_title = yes
 	
 	replace_order = yes
 	
-	monthly_salary = 0
-	monthly_prestige = 1.50
+	monthly_salary = 5
+	monthly_prestige = 5
```

Comment:
This is indeed the player granting a minor title to someone else, named "Gengis"